torch/jit/__init__.py

```diff
@@ -1653,6 +1653,20 @@ def save(self, *args, **kwargs):
             """
             return self._c.save(*args, **kwargs)
 
+        def save_for_mobile(self, *args, **kwargs):
```

Comment:
Should this be part of the public API for ScriptModule? I thought we were not exposing the lite interpreter externally, so this may be confusing for external users.

Reply:
I thought the "_" is needed in C++ only. I might missed it here. Just to confirm: should we rename it to "_save_for_mobile"? I could have a quick patch for it.